extensions/wikia/ArticleComments/modules/templates/ArticleComments_Comment.php

```diff
@@ -1,5 +1,5 @@
 <? if ( is_array( $comment ) ) :?>
-<li id="comm-<?=$commentId?>" class="comment SpeechBubble <?=$rowClass?>" data-user="<?=$comment['username']?>">
+<li id="comm-<?=$commentId?>" class="comment SpeechBubble <?=$rowClass?><?= $comment['isStaff'] ? 'staff' : '' ?>" data-user="<?=$comment['username']?>">
```

Comment:
There should be a space before `"staff"`, otherwise it will be merged with a value of `$rowClass` variable.